src/devices/tuya.ts

```diff
@@ -9911,6 +9960,7 @@ export const definitions: DefinitionWithExtend[] = [
         },
     },
     {
+        zigbeeModel: ["ZG-204ZL"],
```

Comment:
I don't think these devices have this zigbee model? (note that this is different from the `model`)

Reply:
hello，Hello, the ZIGBEE module model of the ZG-204Z/ZG-204ZL products we will produce in the future is not Tuya's model.
![abcedf](https://github.com/user-attachments/assets/b36fb232-875d-4717-8cbe-95abf5ae594d)